code/datums/abnormality/_ego_datum/waw.dm

```diff
@@ -264,6 +264,22 @@
 	item_path = /obj/item/clothing/suit/armor/ego_gear/spore
 	cost = 50
 
+// Servant of Wrath - Blind Rage
+
+/datum/ego_datum/weapon/blind_rage
+	item_path = /obj/item/ego_weapon/blind_rage
+	cost = 50
+
+/datum/ego_datum/armor/blind_rage
+	item_path = /obj/item/clothing/suit/armor/ego_gear/blind_rage
+	cost = 50
+
+/*
+/datum/ego_datum/weapon/blind_gun
```

Comment:
If you are going to leave the weapon in the code, might as well leave the datum too so that it's at least obtainable through the wishing well.

Reply:
Of course, if you do this please remove its use of the burning debuff that turns people into hermit staffs, or change it to use its own version of the debuff. Failing that, you could also make use of the DoT code used by adoration instead of a debuff.